extensions/amp-ad-network-gmossp-impl/0.1/amp-ad-network-gmossp-impl.js

```diff
@@ -49,7 +51,8 @@ export class AmpAdNetworkGmosspImpl extends AmpA4A {
 
   /** @override */
   getAdUrl() {
-    return this.element.getAttribute('src');
+    return this.element.getAttribute('src').replace(GMOSSP_BASE_URL_,
+        GMOSSP_BASE_A4A_URL_);
```

Comment:
I noticed that isValidElement expects the old url format.  Do you want to update to also allow a4a path?